arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -658,6 +658,14 @@
 					<string>NDVIRaster</string>
 				</array>
 			</dict>
+			<dict>
+				<key>descriptionText</key>
+				<string>Edit the values of a KML ground overlay.</string>
+				<key>storyboardName</key>
+				<string>EditKMLGroundOverlay</string>
+				<key>displayName</key>
+				<string>Edit KML ground overlay</string>
+			</dict>
```

Comment:
If you put this ahead of `Apply raster rendering rule`, please also adjust its order in the project file.